src/main/java/eu/enexa/docker/ContainerManagerImpl.java

```diff
@@ -156,10 +154,27 @@ public String startContainer(String image, String containerName, List<AbstractMa
             // add extra requirment based on the image name
             dockerHostConfig = addExceptionalConditions(image, allBinds, dockerHostConfig);
 
+            Map<String, String> labels = new HashMap<>();
+            labels.put("traefik.enable","true");
+            labels.put("traefik.http.routers."+containerName+".rule","Host(\"enexa-demo.cs.uni-paderborn.de\") && PathPrefix(\"/"+containerName+"\")");
```

Comment:
`enexa-demo.cs.uni-paderborn.de` should be configurable, i.e., come from outside of this class.

The whole set of traefik labels should only be added, if the host URL of traefik is defined.

Reply:
done